src/actions/oldInvitationsActions.js

```diff
@@ -142,7 +142,9 @@ export const fetchOldInviteNotificationsAction = () => {
     });
 
     await dispatch(updateConnectionsAction());
-    dispatch(getExistingChatsAction());
+    await dispatch(getExistingChatsAction());
+
+    return Promise.resolve(true);
```

Comment:
why do we need this? (just curious)

Reply:
during debug, it ran async even though await was called before the dispatch